nomenclature/definitions/variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,86 @@
 # List of variables related to the electricity grid
+# Variables related to electricity demand
+Network|Electricity|Value of Lost Load:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
 
-Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+Network|Electricity|Demand not served:
+   description: Demand-not-served.
+   unit: MWh
+
+Network|Electricity|Reserve|Requirement|Frequency Containment:
+   description: Demand in frequency containment reserve (FCR)
    unit: MW
-   
-Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+
+Network|Electricity|Reserve|Requirement|Automatic Frequency Restoration:
+   description: Demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
-Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+
+Network|Electricity|Requirement|Inertia:
+   description: Demand in inertia
    unit: second
-   
-Maximum Flow|Electricity|Grid:
-   description: maximum flow of an interconnection between 2 regions
-   unit: MW
-   
-Minimum Flow|Electricity|Grid:
-   description: minimum flow of an interconnection between 2 regions
+
+# Variables related to transmission lines
+Network|Electricity|Active Power Flow:
+   description: Active power flow through a line
    unit: MW
-   
-Impedance|Electricity|Grid:
-   description: impedance of an interconnection between 2 regions
+
+Network|Electricity|Impedance:
+   description: Impedance of an interconnection between 2 regions
    unit: S
+
+Network|Electricity|Investment:
+   description: Overnight investment (capital) cost in transmission lines of the power network
+   unit: [million EUR/yr, billion US$2010/yr]
+
+Network|Electricity|Fixed Charge Rate:
+   description: Fixed charge rate to annualize the overnight investment cost. The Fixed Charge Rate (FCR) is the percentage of the total investment cost that is required over the project life per year to cover the minimal annual revenue requirements.
+   unit:
+
+Network|Electricity|Line Type:
+   description: Line type related to alternating current (AC) and direct current (DC)
+   unit: [AC, DC]
```

Comment:
I don't think that the line type works as a variable. If I understand correctly how this is intended, it is not the **unit** that is AC or DC, but the **value** of the timeseries - but non-numerical values don't work with the data format. I would remove this entry for now, and start a discussion (as a new issue( on how to include that information in another place.